build/main.js

```diff
@@ -14,8 +14,9 @@
 
 require('babel-polyfill');
 require('babel-register')();
-
+const semver = require('semver');
```

Comment:
nit: I'd like a newline here, but this is a personal preference so whatever you want.